### DISCUSSION TITLE
How to use flexboxes ?

### DISCUSSION THREAD

**Discussion by Ulala01** [Ulala01/Project-Y#7] [Q&A] [answered]
Can anyone help me with using flexboxes and explain to me what are they?

**Comment by tsaxena4k** [comment, marked as answer]
# CSS Flexbox Layout Module

Before the Flexbox Layout module, there were four layout modes:

- Block, for sections in a webpage
- Inline, for text
- Table, for two-dimensional table data
- Positioned, for explicit position of an element

The Flexible Box Layout Module, makes it easier to design flexible responsive layout structure without using float or positioning.
